MaxText/configs/base.yml

```diff
@@ -103,6 +103,10 @@ jax_cache_dir: "~/jax_cache"
 # Hardware
 hardware: 'tpu' # Supported hardware types are 'tpu', 'gpu' and 'cpu'
 
+# Jax Initilization configuration
+process_id: -1
```

Comment:
Can't we get rid of this?

Reply:
@rwitten I opted not to set them as `envvars`  because they are not global across all processes. Specifically, the values for process_id and local_device_id are specific to each process, whereas values like `COORDINATOR_PORT` and `COORDINATOR_IP` are more appropriately configured as `envvars` since they apply globally to all processes.

Resolving for now please reopen if you have a different perspective.

Reply:
Sorry I missed this the first time. Theres a typo in `Initilization` and let's add a comment saying that this is only environments such as GPUs where we operate one process per device so an env variable is unhelpful. (P.S. you could still use an env var and that would still be preferable! You'd just export them before launching the process. But now I understand your goal enough that I think I'm ok with this...)

Reply:
Ack. I have passed it as `envvar` to make it more `Jax-y`